packages/@okta/vuepress-site/docs/release-notes/2024/index.md

```diff
@@ -4,6 +4,59 @@ title: Okta API Products release notes 2024
 
 > Help us improve our release notes by filling out this short [survey](https://surveys.okta.com/jfe/form/SV_4VEZcIGOX0TBgkC).
 
+## February
+
+### Monthly release 2024.02.0
+
+| Change | Expected in Preview Orgs |
+|--------|--------------------------|
+| [Assign admin roles to an app](#assign-admin-roles-to-an-app)| June 14, 2023 |
+| [DPoP support for Okta management APIs is GA in Production](#dpop-support-for-okta-management-apis-is-ga-in-production) | December 13, 2024 |
```

Comment:
just confirming this is supposed to be Dec 2024

Reply:
yes